WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Package Details/Multi-package/ShippingLabelHazmatCategory.swift

```diff
@@ -131,7 +131,7 @@ extension ShippingLabelHazmatCategory {
         static let division41 = NSLocalizedString("Division 4.1 – Mailable flammable solids and Safety Matches Package - Safety/strike on box matches, " +
                                                   "book matches, mailable flammable solids",
                                                   comment: "A hazardous material description stating when a package can fit into this category")
-        static let division51 = NSLocalizedString("Division 5.1 – Oxidizers Package - Hydrogen peroxide (8 to 20% concentration)",
+        static let division51 = NSLocalizedString("Division 5.1 – Oxidizers Package - Hydrogen peroxide (8 to 20％ concentration)",
```

Comment:
😉 

There's still the risk that some translators will use the real `%` character in their translations, and if it happens that the word for `concentration` in those languages start with `c`,`i`, `d`, `s`, `f`, `x`, `p`, or any valid placeholder of the fprintf spec, that will still cause the linter to complain with a false positive on placeholder mismatch.

We could add something about it in the `comment` of that `NSLocalizedString`… though it could end up quite hard to explain this in a concise way in the comment (as in "please don't use the `%` in your translations but instead either keep the FULL-WIDTH PERCENT character or use any other variation (e.g. using `٪` aka the ARABIC PERCENT SIGN U+066A for the arabic translation is OK)"… quite a mouthful for a comment that we aren't even sure all translator really read 😅 

So I guess the best option is to 🤞  that they won't use `%` + valid placeholder character, and if they do you will then probably just fix the translation in GlotPress yourself and submit+validate it before re-exporting…